src/test/cli/rbd/help.t

```diff
@@ -5,7 +5,7 @@
   
   Positional arguments:
     <command>
-      bench-write                 Simple write benchmark.
```

Comment:
This remove (and below) should also be moved to "make bench-write invisible to user" commit.